druid/src/lens/lens.rs

```diff
@@ -509,3 +537,23 @@ impl<T> Lens<T, ()> for Unit<T> {
         f(&mut ())
     }
 }
+
+#[derive(Debug, Copy, Clone)]
+pub struct Not;
```

Comment:
This should still get docs, I'm surprised we aren't failing CI? we have `#[warn(missing_docs)]` turned on, and one of our CI steps should deny warnings... 🤔